### PR TITLE
[T4A6][T10-B2] Low Yong Cheng

### PR DESCRIPTION
**Pull request by yongchenglow**
Added UtilsTest

### CODE REVIEW

**Comment by yongchenglow** [comment]
Ready for Review

File: src/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
```
Comment (nirandiw) [inline src/seedu/addressbook/common/UtilsTest.java:8]: Add header comments

File: src/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+public class UtilsTest {
+
+	private Object nullObject;
+	private Object twinObjectOne = "twin";
+	private Object twinObjectTwo = "twin";
+	private Object uniqueObject = "unique";
+	private ArrayList<Object> numbers = new ArrayList<Object>();
+	
+	@Test
+	public void isNull(){
```
Comment (nirandiw) [inline src/seedu/addressbook/common/UtilsTest.java:18]: This is not the standard to name the test case. It should be MethodTested_ScenarioTest_returnValue.

i.e., isAnyNull_emptyList_falseReturned

**Comment by nirandiw** [comment]
@yongchenglow Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**